README.md

```diff
@@ -10,5 +10,10 @@ Movie Info ios App using The Movie DB API with MVC to MVVM refactoring
 - Install Xcode 10
 - Get the API Key from https://developers.themoviedb.org/3/
 - Put API Key in MovieService.swift apiKey constant
-- pod install
+- Open terminal
+- set your project path on terminal
+- command : pod init
```

Comment:
It's better to wrap commands by backsticks, so the result will be like this:
`pod init`